mrmustard/math/__init__.py

```diff
@@ -53,8 +53,8 @@ class Math:
     def __getattribute__(self, name):
         if settings.BACKEND == "tensorflow":
             return object.__getattribute__(TFMath(), name)
-        elif settings.BACKEND == "torch":
-            return object.__getattribute__(TorchMath(), name)
+        # elif settings.BACKEND == "torch":
+        #     return object.__getattribute__(TorchMath(), name)
```

Comment:
yeah I don't know why we still leave that there... 😅

Reply:
Somehow, I got errors about it when testing the PyJulia cod. I was planning to uncomment it again later.